m4a_hq_mixer.s

```diff
@@ -17,6 +17,9 @@
 
     /* NO USER SERVICABLE CODE BELOW HERE! YOU HAVE BEEN WARNED */
 
+    .syntax unified
+    .cpu arm7tdmi
```

Comment:
I have never used the `.cpu` directive. Does this make any difference in terms of generated code?

Reply:
It doesn't change the instructions generated, but it will cause it to error if any incompatible instructions are used.